src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.html

```diff
@@ -1,4 +1,5 @@
-<div *ngIf="isLoginActive">
+<div class="container"
+     *ngIf="isLoginActive">
```

Comment:
Don't think this is still necessary
```suggestion
<div *ngIf="isLoginActive">
```

Reply:
I added it there to make it less painful when I am doing the downstream branding. So I intended it to be sort of cleanup.

Reply:
Ok, though it was a remnant from the previous version.

Reply:
Yeah. But it being there will be a benefit so thought I'll just put it there. Hope its okay.